src/main/resources/assets/css/main.less

```diff
@@ -95,6 +95,14 @@ a,
   color: #fff;
 }
 
+.close-input-btn {
```

Comment:
I am not sure how this goes conform with Bootstrap. 
I think the Bootstrap symbols are not included, you've said? So that's ok here..
But we should do it with Bootstrap in the future and should refactor it then.